### PR TITLE
Feat: Add the callback modify query using for the export native action to be able to filter the query

### PR DESCRIPTION
**Pull request by vincent-tarrit**
## Description

- [X] Visual changes (if any) are shown using screenshots/recordings of before and after.

## Code style

- [X] `composer cs` command has been run.

## Testing

- [ ] Changes have been tested.

## Documentation

- [ ] Documentation is up-to-date.

### CODE REVIEW

File: packages/actions/src/Concerns/CanExportRecords.php [hunk 1]
```diff
@@ -367,4 +378,11 @@ public function getFormats(): ?array
     {
         return $this->evaluate($this->formats);
     }
+
+    public function modifyQueryUsing(Closure $callback): static
```
Comment (danharrin) [inline packages/actions/src/Concerns/CanExportRecords.php]:
```suggestion
    public function modifyQueryUsing(?Closure $callback): static
```

**Comment by danharrin** [comment]
Thanks

**Comment by vincent-tarrit** [comment]
Hello, I see that you removed the data from the evaluation. The main idea of this feature was to use the data of the form in the query. Is it possible to put them back ?

**Comment by iprastha** [comment]
> Hello, I see that you removed the data from the evaluation. The main idea of this feature was to use the data of the form in the query. Is it possible to put them back ?

Are there any method or workaround for options to be passed into the query?

**Pull request merged**